src/BuildScriptGenerator/DotNetCore/DotNetCoreBashBuildSnippet.sh.tpl

```diff
@@ -1,25 +1,56 @@
+tmpOutputDir="/tmp/puboutput"
+zippedOutputFileName=oryx_output.tar.gz
+
+outputDir="$DESTINATION_DIR"
 if [ "$SOURCE_DIR" == "$DESTINATION_DIR" ]
 then
 	outputDir="{{ PublishDirectory }}"
-	if [ -d "$outputDir" ]
-	then
-		echo "Destination directory '$outputDir' already exists. Deleting it ..."
-		rm -rf "$outputDir"
-	fi
-else
-	outputDir="$DESTINATION_DIR"
 fi
 
 echo
 dotnetCoreVersion=$(dotnet --version)
 echo ".NET Core Version: $dotnetCoreVersion"
 
 echo
-echo Restoring packages ...
+echo Restoring packages...
 echo
 dotnet restore "{{ ProjectFile }}"
 
-echo
-echo "Publishing to directory $outputDir..."
-echo
-dotnet publish "{{ ProjectFile }}" -c Release -o "$outputDir"
+{{ if ZipAllOutput }}
```

Comment:
Can't we have this zip all output control only in the base script? That is, each snippet controls how it produces the output, but in the end there's a single piece of code that zips all of it?

Reply:
That's a good point. Will look into it.